tunnel.go

```diff
@@ -137,6 +130,46 @@ func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, con
 	return t, nil
 }
 
+// NewTunnel creates a Tunnel client with given options.
+func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, config *Config) (*Tunnel, error) {
+	m, err := nkn.NewMultiClient(account, identifier, config.NumSubClients, config.OriginalClient, config.ClientConfig)
+	if err != nil {
+		return nil, err
+	}
+	<-m.OnConnect.C
+
+	return newTunnel(account, from, to, tuna, config, m)
+}
+
+type FromTo struct {
+	From string
+	To   string
+}
+
+// NewTunnels creates multi tunnel clients with given options.
+func NewTunnels(account *nkn.Account, identifier string, fromTo []FromTo, tuna bool, config *Config) ([]*Tunnel, error) {
+	m, err := nkn.NewMultiClient(account, identifier, config.NumSubClients, config.OriginalClient, config.ClientConfig)
+	if err != nil {
+		return nil, err
+	}
+	<-m.OnConnect.C
+
+	tunnels := make([]*Tunnel, 0)
+	for _, ft := range fromTo {
+		t, err := newTunnel(account, ft.From, ft.To, tuna, config, m)
+		if err == nil {
+			tunnels = append(tunnels, t)
+			if len(ft.From) == 0 || strings.ToLower(ft.From) == "nkn" { // one listener
+				break
+			}
+		} else {
+			log.Printf("newTunnel from %v to %v err: %v", ft.From, ft.To, err)
```

Comment:
I think we should return error if any newTunnel returns error

Reply:
Yes, modified.